styx-e2e-test/src/test/java/com/spotify/styx/e2e_tests/EndToEndTestBase.java

```diff
@@ -110,7 +110,7 @@ public class EndToEndTestBase {
 
   private static final String FLYTE_DOMAIN = "development";
   private static final String FLYTE_LAUNCH_PLAN_NAME = "morning_greeting";
-  private static final String FLYTE_LAUNCH_PLAN_VERSION = "04e346ca5f43fc2778259d77a2d9f64ce42b2a27";
+  private static final String FLYTE_LAUNCH_PLAN_VERSION = "v0";
```

Comment:
Let's restore this change?

Reply:
sure, but these tasks do not exist right now